evm-adapters/src/sputnik/evm.rs

```diff
@@ -211,6 +211,11 @@ pub mod helpers {
     >;
 
     static CFG: Lazy<Config> = Lazy::new(Config::london);
+    static CFG_NO_LMT: Lazy<Config> = Lazy::new(|| {
+        let mut cfg = Config::london();
+        cfg.create_contract_limit = None;
+        cfg
+    });
```

Comment:
Given how the PR ended up going, is this still needed? Think there should be no change required in the VM set up steps?

Reply:
cheatcodes.sol passes the 24kb limit now, which causes the ffi and cheatcode tests to fail otherwise
<img width="1366" alt="Screen Shot 2021-12-26 at 7 25 11 AM" src="https://user-images.githubusercontent.com/31553173/147411051-a2a6ca07-b5b5-4775-8cf0-e40a368eca30.png">
e

Reply:
This has nothing to do with the actual implementation, its just that `Cheatcodes.sol` is large now